pages/community.ftl

```diff
@@ -8,7 +8,8 @@
 
         <ul class="ms-4 mt-2">
             <li><a href="search.html">Search</a> for information in the documentation and mailing list</li>
-            <li><a href="https://keycloak.discourse.group/">Forum</a> where you can ask questions</li>
+            <li><a href="https://github.com/keycloak/keycloak/discussions">GitHub Discussions Forum</a> for discussions and asking questions</li>
+            <li><a href="https://keycloak.discourse.group/">Discourse Forum</a> where you can ask questions</li>
```

Comment:
@DGuhr I'd remove Discourse, considering that is not something we actively maintain, but pretty much a community discussion space.

Reply:
Would also be fine for me, wdyt @stianst ?

Reply:
My 2 cents: I am not 100% sure about removing Discourse. If community already uses it for asking questions and people from community answer questions here, then it will be good to still keep it even if there are not much RH folks answering the questions. (note that Keycloak is community project).

In a similar way, it is questionable if it is good to keep the `and asking questions` in the part related to GitHub Discussions. Shouldn't be the GH discussions main channel for the design discussions and discourse for the questions?

+1 that this should be rather discussed outside of this PR